src/app/modules/item/components/item-dependencies/item-dependencies.component.ts

```diff
@@ -44,10 +45,12 @@ export class ItemDependenciesComponent implements OnChanges, OnDestroy {
   });
 
   createInProgress = false;
+  removeInProgress = false;
```

Comment:
You could have "operationInProgress" "changeInProgress" or "ongoingChange" to have one single variable for both cases. Would make more sense as they are exclusive.

Reply:
Fixed